plugin/cmd/kubectl/bind-service/bind-service.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	v1alpha1 "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1alpha1"
+	clientset "github.com/kubernetes-incubator/service-catalog/pkg/client/clientset_generated/clientset"
+	"github.com/kubernetes-incubator/service-catalog/plugin/cmd/kubectl/utils"
+
+	"k8s.io/client-go/pkg/api/v1"
+	"k8s.io/client-go/rest"
+)
+
+const USAGE = `Usage:
+  kubectl plugin bind-service INSTANCE_NAME BINDING_NAME NAMESPACE`
```

Comment:
The NAMESPACE you want here is not the global `--namespace` flag but one specific to this command, correct? 

Just mentioning because you can find the value from the global flag in the `KUBECTL_PLUGINS_GLOBAL_FLAG_NAMESPACE` and there's also `KUBECTL_PLUGINS_CURRENT_NAMESPACE` which will given the "namespace in use" concept (the one coming from the kubeconfig, or overriden by the `--namespace` flag, etc).